### ISSUE TITLE
Accessibility : Aztec Editor TalkBack Improvements

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10893]
For this audit the governing principles are as follows ([inspired by a previous audit](https://github.com/wordpress-mobile/WordPress-Android/issues/7288))

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.

The following is a list of opportunities to improve the accessibility of the Editor. 

### TalkBack Issues

* When the Title block in the editor is hovered over for the first time it says "Title, EditBox in pager" I think it's referring to the ViewPager but the user won't what that is, this should possibly be renamed to Editor. 
* When you close the keyboard it says "Back button out of pager", related to the above.

**Comment by shiki** [comment]
Good audit! Another thing I noticed is that these buttons are considered as switches even though they are just buttons. 

<img src="https://user-images.githubusercontent.com/198826/70104629-ddc6f680-15fb-11ea-8625-6fbea7e9c0e6.png" width="320">

**Comment by jd-alexander** [comment]
Thanks for raising this point. This exists for all the formatting controls as well. They are all being read as switches. Will investigate.

**Comment by shiki** [comment]
The formatting controls make sense because they can be turned on and off. 🙂

**Comment by jd-alexander** [comment]
> The formatting controls make sense because they can be turned on and off. 🙂
 That makes sense 😅

**Comment by jd-alexander** [comment]
@shiki something else I just noticed was that the "List Switch" isn't actually a switch. It has a spinner that pops up where you make the selection for either a "Unordered List" or a "Ordered List" so should that still be read as a switch? The good thing is when you tap to toggle it announces the options available for selections and notifies you that it's also a popup. Any thoughts?

**Comment by jd-alexander** [comment]
This behavior is also the same for the Link option as well where it plays the role of displaying another popup component.
